src/adapters/rubicon.js

```diff
@@ -178,9 +260,17 @@ function RubiconAdapter() {
       bid.creative_id = ad.ad_id;
       bid.bidderCode = bidRequest.bidder;
       bid.cpm = ad.cpm || 0;
-      bid.ad = _renderCreative(ad.script, ad.impression_id);
-      [bid.width, bid.height] = sizeMap[ad.size_id].split('x').map(num => Number(num));
-      bid.dealId = ad.deal;
+      if (bidRequest.mediaType === 'video') {
+        bid.width = bidRequest.params.video.playerWidth;
+        bid.height = bidRequest.params.video.playerHeight;
+        bid.vastUrl = ad.creative_depot_url;
+        bid.impression_id = ad.impression_id;
+        bid.dealId = ad.deal;
```

Comment:
dealId can come outside - other ones are video specific and height and width mean different things in video.